src/ucp/api/ucp.h

```diff
@@ -1609,6 +1622,67 @@ ucs_status_t ucp_ep_create(ucp_worker_h worker, const ucp_ep_params_t *params,
                            ucp_ep_h *ep_p);
 
 
+/**
+ * @ingroup UCP_ENDPOINT
+ * @brief Add user defined callback for active message.
+ *
+ * This routine adds a user defined callback to be used for ucp_am_send_nb.
```

Comment:
How about something like this:
`This routine installs a user defined callback to handle incoming active messages with a specific id. This callback is called whenever an active message, which was sent from the remote peer by @ref for ucp_am_send_nb, is received on this worker.`